mlrun/config.py

```diff
@@ -140,6 +140,9 @@
                 "enrichment_mode": "disabled",
                 # default 65534 (nogroup), set to -1 to use the user unix id
                 "enrichment_group_id": 65534,
+                "pipelines": {
+                    "kfp_pod_user_id": None,
```

Comment:
Set it to 5 which is the default user unix id  for pipelines as this is not expected to be enriched using our provisioner system.
```suggestion
                    "kfp_pod_user_unix_id": 5,
```